.github/renovate.json5

```diff
@@ -14,12 +14,24 @@
     rangeStrategy: "update-lockfile",
   },
   pep621: {
+    // The default for this package manager is to only search for `pyproject.toml` files
+    // found at the repository root: https://docs.renovatebot.com/modules/manager/pep621/#file-matching
     fileMatch: ["^(python|scripts)/.*pyproject\\.toml$"],
   },
   pip_requirements: {
-    fileMatch: ["^docs/requirements.*\\.txt$"],
+    // The default for this package manager is to run on all requirements.txt files:
+    // https://docs.renovatebot.com/modules/manager/pip_requirements/#file-matching
+    // `fileMatch` doesn't work for excluding files; to exclude `requirements.txt` files
+    // outside the `doc/` directory, we instead have to use `ignorePaths` with a "negative glob pattern".
+    // See:
+    // - https://docs.renovatebot.com/modules/manager/#ignoring-files-that-match-the-default-filematch
+    // - https://docs.renovatebot.com/configuration-options/#ignorepaths
+    // - https://docs.renovatebot.com/string-pattern-matching/#negative-matching
+    ignorePaths: ["!docs/**"]
```

Comment:
I haven't looked at the docs but do you think it's still reasonable to keep the `requirements.*\\.txt` part?

Reply:
I guess the best way to verify this would be to merge and check the dependency dashboard issue?

Reply:
> I haven't looked at the docs but do you think it's still reasonable to keep the `requirements.*\\.txt` part?

The double negative of "ignoring a negative glob pattern" is kind of melting my brain. But I _think_ that makes sense, yes -- thanks!

Reply:
It's also so strange to me that `fileMatch` needs to be a regex whereas `ignorePaths` needs to be a glob